tests/test.rfb.js

```diff
@@ -23,18 +23,18 @@ import sinon from '../vendor/sinon.js';
     window.UIEvent = UIEvent;
 })();
 
-const push8 = function (arr, num) {
+const push8 = (arr, num) => {
```

Comment:
Wouldn't it be better to turn these in to normal functions instead?

Reply:
What do you mean?

`push8(arr, num)` // Not possible since it's not in a function
`const push8 = function (arr, num) {` Zero advantages since it´s a pure function.
`function push8(arr, num) {` I prefer this notation but I tried to minimize the changes.

Reply:
The last one. You already had a bunch of such transformations, so we've already started down that path.